1-install-vcpkg.bat

```diff
@@ -0,0 +1 @@
+vcpkg.exe install boost:x64-windows fftw3:x64-windows gettext:x64-windows glibmm:x64-windows gtkmm:x64-windows libxmlpp:x64-windows libsigcpp:x64-windows pango:x64-windows sdl2-mixer:x64-windows zlib:x64-windows

```

Comment:
I suggest to use `%VCPKG_DEFAULT_TRIPLET%` environment variable, so we can easily switch between them.
```
set VCPKG_DEFAULT_TRIPLET=x64-windows
c:/projects/vcpkg/vcpkg.exe install boost-odeint fftw3 gettext glibmm gtkmm libxmlpp libsigcpp pango sdl2-mixer zlib
```

Also we don't need the full BOOST installation, only Boost.odeint.